packages/chain-adapters/package.json

```diff
@@ -44,11 +44,11 @@
   },
   "devDependencies": {
     "@shapeshiftoss/caip": "^1.11.0",
-    "@shapeshiftoss/hdwallet-core": "^1.18.2",
-    "@shapeshiftoss/hdwallet-native": "^1.18.2",
+    "@shapeshiftoss/hdwallet-core": "^1.19.0",
+    "@shapeshiftoss/hdwallet-native": "^1.19.0",
     "@shapeshiftoss/types": "^1.23.0",
-    "@shapeshiftoss/unchained-client": "^5.1.0",
-    "@shapeshiftoss/unchained-tx-parser": "^5.1.0",
+    "@shapeshiftoss/unchained-client": "^5.1.1",
+    "@shapeshiftoss/unchained-tx-parser": "^5.1.1",
```

Comment:
We are now on 5.2.0 if you want to sync it up. There is another PR in the pipeline with that change